napari/layers/labels/_tests/test_labels_mouse_bindings.py

```diff
@@ -65,28 +59,36 @@ def test_paint(Event, brush_shape, expected_sum):
 @pytest.mark.parametrize(
     "brush_shape, expected_sum", [("circle", 244), ("square", 274)]
 )
-def test_paint_scale(Event, brush_shape, expected_sum):
+def test_paint_scale(brush_shape, expected_sum):
     """Test painting labels with circle/square brush when scaled."""
     data = np.ones((20, 20))
     layer = Labels(data, scale=(2, 2))
     layer.brush_size = 10
     layer.brush_shape = brush_shape
     layer.mode = 'paint'
     layer.selected_label = 3
-    layer.position = (0, 0)
+    position = (0, 0)
 
     # Simulate click
-    event = ReadOnlyWrapper(Event(type='mouse_press', is_dragging=False))
+    event = ReadOnlyWrapper(
+        CursorEvent(data_position=position, type='mouse_press')
+    )
```

Comment:
same as before, maybe ReadOnlyWrapper unnecesary if RO model or frozen dataclass.

Reply:
The current code updates the `.__wrapped__` property here and I'm not quite sure how to achieve the same functionality without that https://github.com/napari/napari/blob/d5a28122129e6eae0f5ada77cb62c4bd5a714b60/napari/utils/interactions.py#L146 otherwise I would drop it too